src/pms-spec.yaml

```diff
@@ -12,21 +12,7 @@ info:
     name: MIT
 
 servers:
-  - url: "{protocol}://{ip}:{port}"
-    description: The full address of your Plex Server
-    variables:
-      protocol:
-        default: "https"
-        description: The protocol to use for the server connection
-        enum:
-          - "http"
-          - "https"
-      ip:
-        default: "10.10.10.47"
-        description: The IP address or hostname of your Plex Server
-      port:
-        default: "32400"
-        description: The port of your Plex Server
+  - $ref: "./servers/dynamic-server.yaml"
```

Comment:
This appears to be illegal OpenAPI, and will be a blocker for this PR

Reply:
Yes, I have reverted it. Apart from being illegal, it didn't add much